cmd/wallet/README.md

```diff
@@ -0,0 +1,19 @@
+WALLET
+======
+
+## IMPORTANT:
+
+### This software is for TESTING ONLY. Do NOT use it for handling real money.
+
+`wallet` is a simple, no-frills wallet software operated via the command line.\
+It is capable of generating wallet key-pairs, printing a wallet's current balance, and sending simple transactions.
+
+
+Usage
+-----
+
+* Create a new wallet key-pair: `wallet create --testnet`
+* Print a wallet's current
+  balance: `wallet balance --testnet --address=kaspatest:000000000000000000000000000000000000000000`
```

Comment:
The formatting is a bit weird here and in the next bullet...
I'd expect the line break to be after `balance: `